### ISSUE TITLE
REVIEW OF GDSProject Contract

### ISSUE THREAD

**Issue by tompet815** [Englund0110/GDSProject#1]
## Introduction
Fine

## Vision
Well described. Easy to understand.

### Use Case diagram
Is this really only meant for the employees? - As the usecase describe, its only the Employee there are going to have all the access to the system - with all the given functionality. Isn't there any other users than that? - Seems like there is in the general description of the system (Third party - Customers maybe?)- This should be clear. 

### Use Case description
Fine use case description - Still the actor comment from above. 

## Domain Model
* Domain model contains attribute type, for example, string, enumeration etc., so it seems like Entity model.
* The aggregation arrow between Schedule and Flight, and between Booking and Passenger must be pointing wrong direction? The arrow shows us that a Flight contains may Schedules/ a Passenger contains many Bookings.
* Constraints are nice, easy to understand. Just the expression “(min&max)” is not clear. Min and max of what? 

## Fully Dressed Use Case
The name should be the same as in the usecase diagram.   
You are missing some points in the fully dressed usecase which is:  
* Goal
* Stakeholders interest  

Also, the main success scenary isnt detailed. Which i think is required.   
Example:   
“
1.1 The customer visits the rental company's website.
1.2 The customer picks a location from where he/she wants to rent a car.
1.3 The customer picks a date and time period for the rental.
1.4 The customer is shown a selection of vehicles available for rental at the chosen location.
1.5 The customer picks which model of vehicle to be rented EI model A to F.
1.6 The customer submits the relevant rental information (Driver's license number, Identifying information).
1.7 The information is processed to ensure the drivers license is valid.
1.8 The customer now picks payment options, either pay directly through the website (Credit card or bank transfer) or direct payment at rental company.
1.9 Additional options such as insurance is chosen or left alone.
2.0 Booking is being processed and payment charged (In case of payment through website).
2.1 Booking is completed and receipt shown to customer. 
“
Taken from our contract

## Summing Up
Generally it is a nice contract. Easy to understand. 
Non-functional requirements are missing.
Fully dressed usecase need a little extra work. 
Also it’s a bit confusing if there is only one actor for this system. This should be more clear somehow.  

Good work so far! :)
